### DISCUSSION TITLE
HomeAssistant

### DISCUSSION THREAD

**Discussion by yungthiel** [joni1802/ts3-manager#61] [General]
Did someone try to pull data from the website to create sensors in HomeAssistant?

**Comment by joni1802** [comment]
Can you please describe it more in detail what you want to acomplish. Do you want to pull data from the TS3 Manager via HTTP requests? Like for example pull the client list of all currently connected clients and send it to Home Assistant?

**Reply by yungthiel** [reply]
Yes, exactly I want to display all clients connected to te server, or at least the number of clients.

**Reply by joni1802** [reply]
The TS3 Manager is not the right tool for that because everything is rendered on the client side. So when you are doing a HTTP request to the TS3 Manager your response will be an empty HTML file with no data. 
Use the official WebQuery which was released last year by TeamSpeak. For more infos take a look here
[https://community.teamspeak.com/t/webquery-discussion-help-3-12-0-onwards/7184](https://community.teamspeak.com/t/webquery-discussion-help-3-12-0-onwards/7184)

**Reply by yungthiel** [reply]
Okay thank you!
